sklearn/svm/tests/test_bounds.py

```diff
@@ -74,3 +76,55 @@ def test_ill_posed_min_c():
 def test_unsupported_loss():
     with pytest.raises(ValueError):
         l1_min_c(dense_X, Y1, loss='l1')
+
+
+_MAX_UNSIGNED_INT = 4294967295
+_MAX_INT = 2147483647
+
+
+@pytest.mark.parametrize('seed, val',
+                         [(None, 81),
+                          (0, 54),
+                          (_MAX_UNSIGNED_INT, 9)])
+def test_set_seed(seed, val):
```

Comment:
Since this is located in a file shared with other tests, I recommend to put a specific name and explanation

```suggestion
def test_newrand_seed(seed, val):
    """Test that set_seed_wrap works correctly with bounded_rand_int_wrap"""
```